modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectActionDescriptionException.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Aquiles Duarte
+ */
+public class ObjectActionDescriptionException extends PortalException {
+
+	public List<Object> getArguments() {
+		return _arguments;
+	}
+
+	public String getMessageKey() {
+		return _messageKey;
+	}
+
+	public static class MustBe75Characters
+		extends ObjectActionDescriptionException {
+
+		public MustBe75Characters() {
+			super(
+				Arrays.asList("Description", 75),
+				"Description must be less than 76 characters",
+				"Description cannot exceed 75 characters");
```

Comment:
@Aquiiiles please use the existing language key `only-x-characters-are-allowed-in-the-x-field` 

```java
public MustBeLessThan75Characters() {
	super(
		Arrays.asList(75, "description"),
		"Description must be less than 75 characters",
		"only-x-characters-are-allowed-in-the-x-field");
}
```

See [ObjectActionNameException](https://github.com/carolmariaabb/liferay-portal/blob/c5fb9f0c5f83b838bb9a9faaf90eea9491b7a6c2/modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectActionNameException.java#L29-L34)